Objects/Converters/ConverterGSA/ConverterGSA/ConverterGSA.ToSpeckle.cs

```
@@ -4419,20 +4419,20 @@ private GSAAlignment GetAlignmentFromIndex(int index, GSALayer layer = GSALayer.
 
     #region General Data
 
-    private GSAListType GetListType(string listType)
+    private GSAListType GetListType(ListType type)
     {
-      switch (listType.ToUpper())
+      switch (type)
       {
-        case "NODE":
+        case ListType.Node:
           return GSAListType.Node;
-        case "ELEMENT":
+        case ListType.Element:
           return GSAListType.Element;
-        case "MEMBER":
+        case ListType.Member:
           return GSAListType.Member;
-        case "CASE":
-          return GSAListType.Case;
         default:
-          return GSAListType.Unspecified;
+          return GSAListType.Case;
```

We should have one `case` per value and default should throw an exception. I know we have other switch statements in the Converter project which return an enum value for the default case, but in these scenarios there is logic behind why that particular enum value should be returned by default (ex. returning a NotSet value, or an "other" value or "generic" value, which makes sense in the context of the conversion) - returning `GSAListType.Case` for the default case doesn't make sense for the list conversion.